### DISCUSSION TITLE
Duda sobre almacenamiento de nodos en parte 4 y 5

### DISCUSSION THREAD

**Discussion by gabo1243** [IIC2133-PUC/2024-1#104] [Tarea 1] [answered]
Hola, por ahora estoy ocupando arboles, donde cada nodo tiene una referencia a otro arbol con los mismos valores de x pero ordenados por y, la cosa es que para la 4 hice que al recorrer y buscar por rango se agregaran estos nodos a una lista ligada para así tener un contador al principio de cuantos coincidan e imprimirlo de primero. El tema es que no sé si es válido dado que además de agregarlos, tendré que recorrer esa lista ligada con k elementos  que cumplen el rango para imprimirlos en orden. Como este ultimo recorrido es O(k), no sé si es válido esta forma para cumplir el log(n) en total (igual considero que k es menor a n, por lo que puede no afectar tanto).
Gracias de antemano.

**Comment by sebacarrascop** [comment, marked as answer]
Holaa, esto se explica en las discussions #97 y #102, si no se entiende algo responde a este mensaje ;)
